k2/util/Logging.cc

```diff
@@ -0,0 +1,57 @@
+// $FILE_PATHNAME $HEADER_FILENAME $HEADER_PATHNAME
+
+// Copyright (c) 2020 Xiaomi Corporation ( authors: Meixu Song )
+//
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/util/Logging.h"
+
+#ifdef K2_USE_GLOG
```

Comment:
Why not to use glog directly?  K2 is a training framework and I don't think
mobile devices have the power for training. As far as I know, glog
is supported on Linux, macOS and Windows.

Reply:
Actually, k2 would be used in inference as well.
Of course that will require some things to be worked on!

Reply:
For mobile is one thing, the other more important reason is about glog itself actually. After investigations, I found glog is not beautiful as I thought. Even the logging system itself is a trade-off/confusing concept, that would easily be noisy, expose too much global flags/macro, takes too much responsibility (exception/error handling). So in conclusion, I found there isn't the best practice. However, glog with one minimal glog-like logging is preferred by many great projects. A minimal glog could be very simple to just support minimal functions of glog. And loguru turn out to be the expected one. So, with loguru, the two-logger thought is implemented as this PR.